src/vscode-bicep/src/utils/removePropertiesWithPossibleUserInfo.ts

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+export function removePropertiesWithPossibleUserInfoInDeployParams(
+  value: string
+): string {
+  const deployParamsPattern = new RegExp('.*"token":\\s*"(?<token>.*)"');
```

Comment:
Use `/.*"token":\\s*"(?<token>.*)"/' instead of the RegExp constructor so V8 can cache it. An alternative way is to make the regex a file level constant.

Reply:
Yep, using a regex literal is better.